// @kolkov/angular-editor/esm2015/lib/config.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes} checked by tsc
 */
/**
 * @record
 */
export function CustomClass() { }
function CustomClass_tsickle_Closure_declarations() {
    /** @type {?} */
    CustomClass.prototype.name;
    /** @type {?} */
    CustomClass.prototype.class;
    /** @type {?|undefined} */
    CustomClass.prototype.tag;
}
/**
 * @record
 */
export function AngularEditorConfig() { }
function AngularEditorConfig_tsickle_Closure_declarations() {
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.editable;
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.spellcheck;
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.height;
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.minHeight;
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.width;
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.minWidth;
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.translate;
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.enableToolbar;
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.showToolbar;
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.placeholder;
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.defaultParagraphSeparator;
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.defaultFontName;
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.defaultFontSize;
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.uploadUrl;
    /** @type {?|undefined} */
    AngularEditorConfig.prototype.customClasses;
}
export const /** @type {?} */ angularEditorConfig = {
    editable: true,
    spellcheck: true,
    height: 'auto',
    minHeight: '0',
    width: 'auto',
    minWidth: '0',
    translate: 'yes',
    enableToolbar: true,
    showToolbar: true,
    placeholder: 'Enter text here...',
    defaultParagraphSeparator: '',
    defaultFontName: '',
    defaultFontSize: '',
    uploadUrl: 'v1/image',
};
